docs/tutorials/fundamentals/viewer.md

```diff
@@ -97,60 +79,76 @@ nbscreenshot(viewer, alt_text="photograph of an astronaut in napari viewer")
 viewer.close()
 ```
 
-Both the {func}`imshow<napari.imshow>` and the {meth}`add_image<napari.components.viewer_model.ViewerModel.add_image>`
-methods accept any numpy-array like
-object as an input, including n-dimensional arrays. For more information on
-adding images to the viewer see the [image layer guide](../../howtos/layers/image).
-Now we will continue exploring the rest of the viewer.
+ {func}`imshow<napari.imshow>` and the {meth}`add_image<napari.components.viewer_model.ViewerModel.add_image>` methods accept any numpy-array like object as input, including n-dimensional arrays. For more information on adding images to the viewer see the [image layer guide](../../howtos/layers/image). 
 
-+++
+Now we will continue exploring the rest of the viewer.
 
 ## Layout of the viewer
 
-The viewer is organized into a few key areas:
+The viewer is organized into a few key areas which are explained in the next sections:
+* Main Menu (top bar menu)
+* Layer Controls
+* Layer Buttons
+* Layer List
+* Viewer Buttons
+* Status Bar
+* Canvas
+* Dimension Sliders
+* Scroll Buttons
+* Frame Playback  
+
+The image below has the areas of the viewer labeled: 
 
-![image: viewer layout](../assets/tutorials/viewer_layout.jpg)
+  ![image: viewer layout](../assets/tutorials/Viewer-with-arrows.png)
 
 We'll go through each of these in the next sections.
 
-+++
+### Main menu or top bar menu
 
-### Canvas
+The main menu consists of the **File**, **View**, **Window**, **Plugins**, and **Help** options which are illustrated in this section.  
+
+* **File** contains the options shown in the graphic below: 
+![image: File Menu](../assets/tutorials/main-menu-file.png)  
 
-The canvas is in the center of the viewer and contains the visual display
-of the data passed to **napari**, including images, point, shapes, and our other
-supported data types. Under the hood the canvas is a `vispy.scene.SceneCanvas`
-object which has built-in support for features such as zooming and panning. As
-`vispy` uses `OpenGL` and your graphics card, panning and zooming are highly
-performant. You can also return to the original zoom level by clicking the
-`home` button in the viewer buttons panel.
+    All the options on the **File** menu are self-explanatory except **Preferences**. **Preferences** has the options shown below. To learn more about the **Preferences** menu, visit this [tutorial](https://napari.org/stable/guides/preferences.html).  
+    ![image: Preferences sub-menu](../assets/tutorials/main-menu-file-sub-menu-preferences.png)  
```

Comment:
Thank you for telling me about this issue.  The reason **Preferences** is mentioned is because we talk about setting the theme. In my opinion this whole tutorial is developer centric.  We are just beginning to introduce the GUI here.  A future edition is planned that will separate the GUI technique from the developer centric technique.  But that is just one of many priorities. (No date has been established for it yet.)

Reply:
The theme is mentioned under the heading, Changing the viewer theme.

Reply:
I would still delete. 
I feel that *this* Viewer tutorial is very user-centric, which is good.
That Preferences guide is for devs, so it's really not appropriate to link here as a **tutorial** which suggests it's similar to the current document.
We can explicitly mention the File menu when mentioning the theme in that section (if we already are not).

Reply:
BTW, on macOS preferences are under the napari menu and not File. Not sure about linux.